.circleci/config.yml

```diff
@@ -0,0 +1,242 @@
+version: 2.1
+
+orbs:
+  android: circleci/android@0.2.1
+  react-native: react-native-community/react-native@4.4.2
```

Comment:
react native?

Reply:
Oh I was using some stuff from this orb but not anymore I will remove it